src/main/java/com/assure/movie/model/domain/Actor.java

```diff
@@ -11,7 +11,7 @@
 public class Actor implements Identifiable<Long> {
 
     @Id
```

Comment:
Just a quick question about the annotations in the Entity classes, I see you are using the JPA annotations above of the properties definition, I have seen in other projects that those annotations are above the getter methods.
What is the difference between those ways of using JPA annotations?

Reply:
Both annotations work in the same way, but after research a litle I think the property annotations is better than the getter, due to the following:
- better readibility of the code(is more easy to see the columns in a simple view when is property annotation)
- you don't add getter and setter methods that you do not need to use
- you can use the getter an setter methods in the way that you can because those are not being used by JPA
- you do not need to use the @transient annotation

Reply:
I think the reasons are good, but not forget anyway we will need to create the getter/setter methods, how will you assign a new value to a existing entity if you need to execute an update operation for example?